### ISSUE TITLE
typesettercms.com down (cloudflare)

### ISSUE THREAD

**Issue by agail** [Typesetter/Typesetter#702]
Someone forgot to pay the bill?

**Comment by mckaygerhard** [comment]
the problem is that, is not free.. but typesetter just works, i still use it and works like a charm

**Comment by gtbu** [comment]
For Yoshua : There is a cheaper solution for hosting : A provider like [inet.ws](https://cloud.inet.ws/order/vps/19) offers cheap vservers (8Dollar - around the globe) with Ubuntu up to 2024 with 2TB Traffic.  Together with mod_deflate and fail-to-ban and [wondershaper](https://github.com/magnific0/wondershaper) - BW 1Mb/sec. - cloudflare will be superflous.

**Comment by mckaygerhard** [comment]
> For Yoshua : There is a cheaper solution for hosting : A provider like [inet.ws](https://cloud.inet.ws/order/vps/19) offers cheap vservers (8Dollar - around the globe) with Ubuntu up to 2024 with 2TB Traffic. Together with mod_deflate and fail-to-ban and [wondershaper](https://github.com/magnific0/wondershaper) - BW 1Mb/sec. - cloudflare will be superflous.

nonsense.. sourgeforce provided php+mysql free.. must be use sourgefoce buil in hosting

**Comment by gtbu** [comment]
I dont think that the distribution (with its specific interface etc.)  and the forum can be installed so easy on sourceforge. There are several [cheap providers](https://digital.com/best-apache-hosting/), but most do not allow  to install server-plugins.

**Comment by mckaygerhard** [comment]
> I dont think that the distribution (with its specific interface etc.) and the forum can be installed so easy on sourceforge. There are several [cheap providers](https://digital.com/best-apache-hosting/), but most do not allow to install server-plugins.

ignorance ... i already used typesetter at sourceforge.. plugin manager works last time, and just install the plugin manually when fails

**Comment by gtbu** [comment]
I dont think that the cloudflare - bill is not payed, beause Cloudflare offers free accounts. Maybe the provider offers only newer php-versions, at which the old Typesetter-forum and distribution do not run, and therefore it needs adaptions. 
At the moment I AM the only Typesetter - developer , and if  Joshua wants, i can try to overwhole the code so that it runs on php8.2.

**Comment by unix4you2** [comment]
Hi there!!!
Does anybody know how to install plugins  while typesettercms.com is offline?  We are trying to install plugins but the process fail because the main site is down right now.
Thanks in advance!!

**Comment by gtbu** [comment]
Yes - just install them under   /addons - just like themes under  /themes

This has also the advantage, that You can then install them under "Available themes" and if You uninstall them,  they are just deactiveted and not deleted. Installations from the Web are deleted,if You uninstall them !

**Comment by unix4you2** [comment]
> Yes - just install them under /addons - just like themes under /themes
> 
> This has also the advantage, that You can then install them under "Available themes" and if You uninstall them, they are just deactiveted and not deleted. Installations from the Web are deleted,if You uninstall them !

Thank you @gtbu  and where could I find the plugin download to uncompress in that folder?  because in the https://github.com/Typesetter/Typesetter/tree/master/addons path I can't see the sme plugins taht I could install by web.  Is there anther location to find and download the plugin files?

I mean, where are the sources for this plugins that you can see in tne web installer:
![image](https://github.com/Typesetter/Typesetter/assets/2445412/b8705463-1895-4503-af8c-bc040679e0d5)


Thanks agains for your help.

**Comment by gtbu** [comment]
The plugins installed by web are under data/Addondata and data/addoncode (crypted directory-names ----Themes under /data/themes)

*After You unzipped the specific addon-x.zip into a separate folder under /addons --- You will then see it in the list of available plugins 

*I hope the problems with typesettercms.com/ themes or /addons will be removed until next month (probably deprecated code - and the provider supports only php8)
